### DISCUSSION TITLE
How to find the working directory?

### DISCUSSION THREAD

**Discussion by esemeniuc** [lindell/multi-gitter#245] [Q&A]
After running the script, there should be an option to see the directory that was used. Currently, I am hacking this by using `pwd` as the script

**Comment by lindell** [comment]
Please explain the use-case for this to make it easier to understand 🙂 
Do you mean the directories in which all repositories are cloned? In that case, those directories are removed after the changes are made, to not keep unnecessary data on the users disk.

**Reply by esemeniuc** [reply]
Yes I mean the directory where all the repos are cloned. I thought there were artifacts left behind that could be viewed. It wasn't clear to me that it would be automatically deleted since the README.md doesn't include any keywords like `delete`, `remove`, `clean`, or `prune`

**Comment by klieret** [comment]
How about adding a new option `--keep` for this? I think this would be particularly useful for dry runs where you want to check what has happened.

**Reply by sigma67** [reply]
Created a feature request here: https://github.com/lindell/multi-gitter/issues/493
